empty_things3_inbox_to_notion.py

```diff
@@ -28,17 +28,22 @@ def addParagraphToBlock(block_id, paragraph_content):
 
 
 def addContentToBlock(
-    block_id, content: list, *, padded=True, blank_header=False, as_callouts=False
+    block_id, content: list, *, title="", padded=True, blank_header=False, as_type: tn.BlockTypes=None
 ):
-    if as_callouts:
-        content_ = [tn.create_callout_block(title="", children=content)]
+    if as_type == tn.BlockTypes.CALLOUT:
+        content_ = [tn.create_callout_block(title=title, children=content)]
+    elif as_type == tn.BlockTypes.TOGGLE:
+        content_ = [tn.create_toggle_block(title=title, children=content)]
+    elif as_type:
+        content_ = [tn.create_block(title=title, children=content, type=type)]
     elif padded:
         content_ = [tn.create_paragraph("")] + content + [tn.create_paragraph("")]
         if blank_header and content and not tn.objIsHeader(content[0]):
             content_ = [tn.create_heading("")] + content
     try:
         notion.blocks.children.append(block_id, children=content_)
     except:
+        print(f"failed call to addContentToBlock() where {block_id=} {content_=}")
```

Comment:
this is good -- add more info for error messages is good, don't worry abt it be perfect

- probably should add to docs how to see debug (altho already in workflow and stdout) so OK